packages/unuse-vue/src/index.spec.ts

```
@@ -0,0 +1,196 @@
+
+// eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types
+export function useSetup<V>(setup: () => V) {
+  const Comp = defineComponent({
+    setup,
+    render() {
+      return h('div', []);
+    },
+  });
+
+  return mount(Comp);
+}
```

Biome (1.9.4)

[error] Do not export from a test file.



(lint/suspicious/noExportsInTest)